AUTHORS.txt

```diff
@@ -31,3 +31,7 @@ Rémy RAKIC
 
 Hugo SMETT
     Developper
+
+Morgane ALONSO
+    Developper
+    malonso AT hybird DOT org
```

Comment:
- People are sorted alphabetically => yeah you are the first !
- "Developper / Documentation writer"

Reply:
```suggestion

Morgane ALONSO
    Developer
    malonso AT hybird DOT org
```

(si tu peux corriger la coquille sur le reste du fichier 🙏 )